Backport from 6.X
https://gitlab.nic.cz/knot/knot-resolver/-/merge_requests/1816.patch
Subject: [PATCH] Fix luajit includedir when cross-compiling
The use of --variable=includedir for luajit doesn't include the
root-directory when cross-compiling.
--- a/modules/meson.build
+++ b/modules/meson.build
@@ -32,8 +32,7 @@ integr_tests += [
   ['serve_stale', meson.current_source_dir() / 'serve_stale' / 'test.integr'],
 ]
 
-mod_inc_dir = include_directories('..', '../contrib',
-  luajit.get_pkgconfig_variable('includedir'))
+mod_inc_dir = include_directories('..', '../contrib')
 
 mod_deps = [
   contrib_dep,
